### PR TITLE
🐌 Add homework No.3

### PR DESCRIPTION
**Pull request by MiksuJak**

### CODE REVIEW

**Comment by ptomaszewska** [comment]
Dobrze wykonana praca domowa :)
- brakuje referencji dot. ablation study i kodu modelu
- super, że jasno jest zaznaczone gdzie zaczyna się autorski kod
- dobry pomysł na stworzenie zbiorczych wykresów
- architektury sieci neuronowych można przedstawić jako "bloczki" - jest gotowa funkcja w kerasie
- mogłoby być więcej pogłębionych wniosków - jak np. ten dotyczący softmax
- ciekawie poprowadzona narracja, której wisienką na torcie był nowy _champion_
- można było też np. analizować jakość modeli per klasa i zobaczyć jak to się zmieniało (nieszablonowe podejście jest nadal w cenie)
